decidim-admin/lib/decidim/admin/test/manage_categories_examples.rb

```diff
@@ -32,13 +32,13 @@
         es: "Mi categoría",
         ca: "La meva categoria"
       )
-      fill_in_i18n_editor(
-        :category_description,
-        "#category-description-tabs",
-        en: "Description",
-        es: "Descripción",
-        ca: "Descripció"
-      )
+      #  fill_in_i18n_editor(
+      #   :category_description,
+      #   "#category-description-tabs",
+      #   en: "Description",
+      #   es: "Descripción",
+      #   ca: "Descripció"
+      # )
```

Comment:
```suggestion
```
Same here, this can be removed

Reply:
@microstudi thanks for the suggestions, we have updated the code and pushed new changes. The PR is ready for review.